.github/workflows/My-CI-Pipeline.yml

```diff
@@ -31,19 +31,21 @@ jobs:
           password: ${{ secrets.DOCKERHUB_TOKEN }}    
       - run: docker build --target production --tag daniphilip/hello-world:prod .
       - run: docker push daniphilip/hello-world:prod
+      - run: curl -dH -X POST ${{secrets.WEBHOOK}}
```

Comment:
One nice addition here can be to add the --fail flag to your curl command (or alternatively the --fail-with-body flag) to ensure that curl reports a failure if it sees a 4XX or 5XX response - otherwise curl reports a success because it successfully received a response, even if that response was a failure - which can lead to some confusing pipeline statuses!